build.gradle

```diff
@@ -81,9 +81,9 @@ jar {
     from(configurations.compile) {
         into "lib/"
     }
-    from(sourceSets.main.java) {
-        into "/"
-    }
+//    from(sourceSets.main.java) {
```

Comment:
Not needed

Reply:
Yes I commended out those line. It will be useful when you want to debug the plugin